src/core/contract_address/sierra_contract_address.rs

```diff
@@ -60,14 +63,20 @@ pub fn compute_sierra_class_hash(
     hasher.update(constructors);
 
     // Hash abi
-    let abi = serde_json_pythonic::to_string_pythonic(
-        &contract_class
+    let abi = {
+        let mut buf = Cursor::new(Vec::new());
+        let mut fmt = serde_json::Serializer::with_formatter(&mut buf, PythonJsonFormatter);
+
+        contract_class
             .abi
             .as_ref()
             .ok_or(ContractAddressError::MissingAbi)?
-            .items,
-    )
-    .map_err(|_| ContractAddressError::MissingAbi)?;
+            .items
+            .serialize(&mut fmt)
+            .map_err(|_| ContractAddressError::MissingAbi)?;
+
+        String::from_utf8(buf.into_inner()).unwrap()
```

Comment:
can you remove this unwrap?

Reply:
It should never trigger since it has to be valid JSON because it's encoded using `serde_json::Serializer`. Should I still try to remove it anyways?